op-batcher/batcher/driver.go

```diff
@@ -536,9 +536,9 @@ func (l *BatchSubmitter) receiptsLoop(wg *sync.WaitGroup, receiptsCh chan txmgr.
 }
 
 // throttlingLoop monitors the backlog in bytes we need to make available, and appropriately enables or disables
-// throttling of incoming data prevent the backlog from growing too large. By looping & calling the miner API setter
-// continuously, we ensure the engine currently in use is always going to be reset to the proper throttling settings
-// even in the event of sequencer failover.
+// throttling of incoming data to prevent the backlog from growing too large. We ensure the engine currently in use
+// is always going to be reset to the proper throttling settings (even in the event of sequencer failover) by
+// binding to the activeSequencerChanged channel as well as the pendingBytesUpdated channel.
```

Comment:
I can't find `activeSequencerChanged` in this code or in op-batcher as a whole. Am I missing something?